doc/src/docs/InputOutputReference/01b-InputOutputReference.md

```diff
@@ -17523,8 +17511,340 @@ This output is the average part-load ratio of the heat pump water heater’s com
+
+#### Field: Dead Band Temperature Difference
+
+This numeric field contains the dead band temperature difference in degrees Celsius. The heat pump compressor “cut-in” temperature is defined as the compressor set point temperature defined above minus this dead band temperature difference. The heat pump compressor cycles on when the water temperature in the tank falls below the “cut-in” temperature. The heat pump compressor remains on until the water temperature in the tank rises above the compressor set point (“cut-out”) temperature defined above. The dead band temperature difference must be greater than 0°C and less than or equal to 20°C. If this field is left blank, the default value is 5°C.
+
+In this model, the heat pump water heater’s DX compression system is considered the primary heat source and the water tank’s heater (element or burner) provides supplemental heat as necessary. Therefore, the cut-in temperature for the heat pump compressor (set point minus dead band temperature difference) is usually higher than the set point temperature for the heater (element or burner) in the associated water heater tank object. At times when the water heater tank set point temperature is greater than the cut-in temperature of the heat pump compressor, the heat pump compressor is disabled and the tank’s heater is used to heat the water.
+
+#### Field: Condenser Bottom Location
+
+This numeric field contains the distance from the bottom of the tank to the bottom of the wrapped condenser. 
+
+#### Field: Condenser Top Location
+
+This numeric field contains the distance from the bottom of the tank to the top of the wrapped condenser.
+
+#### Field: Evaporator Air Flow Rate
+
+This numeric field contains the air flow rate across the heat pump’s air coil (evaporator) in cubic meters per second. It is the actual air flow rate to be simulated, which may differ from the rated evaporator air volumetric flow rate specified for the heat pump’s DX coil (Ref. Coil:WaterHeating:AirToWaterHeatPump). Values must be greater than 0 or this field is autocalculatable. If autocalculated (field value = **autocalculate**), the evaporator air flow rate is set equal to the rated heating capacity of the heat pump’s DX coil multiplied by 5.035E-5 m<sup>3</sup>/s/W. When this flow rate is different from the Rated Evaporator Air Volumetric Flow Rate specified in the heat pump’s DX coil object (Ref. Coil:WaterHeating:AirToWaterHeatPump), the user should also specify a Total Heating Capacity Modifier Curve Name (function of air flow fraction) and a Heating COP Modifier Curve Name (function of air flow fraction) in the associated DX coil object to account for differences in capacity and power consumption at the off-rated air flow rate.
```

Comment:
3rd and 7th lines... (Ref. Coil:WaterHeating:AirToWaterHeatPump:Wrapped or ...AirToWaterHeatPump:* as required).